src/components/PeriodicReports/PeriodicReportCard.tsx

```diff
@@ -119,63 +127,72 @@ const PeriodicReportCardInContext = (props: PeriodicReportCardProps) => {
             <Typography color="initial" variant="h4" paragraph>
               {`${type} Reports`}
             </Typography>
-            <div className={classes.relevantReports}>
-              <ReportInfo
-                title="Current"
-                report={dueCurrently}
-                className={classes.relevantReport}
-              />
-              <Divider orientation="vertical" flexItem variant="middle" />
-              <ReportInfo
-                title="Next"
-                report={dueNext}
-                className={classes.relevantReport}
-              />
-            </div>
+            {metRequirement ? (
+              <div className={classes.relevantReports}>
+                <ReportInfo
+                  title="Current"
+                  report={dueCurrently}
+                  className={classes.relevantReport}
+                />
+                <Divider orientation="vertical" flexItem variant="middle" />
+                <ReportInfo
+                  title="Next"
+                  report={dueNext}
+                  className={classes.relevantReport}
+                />
+              </div>
+            ) : (
+              <div>
+                <span>Not all requirements have been met</span> <br />
+                <span>Click here to view remaining list</span>
+              </div>
```

Comment:
Does this have the appropriate typography or does it need to use the `Typography` component?